content/brand/images.md

```diff
@@ -11,7 +11,10 @@ eleventyNavigation:
   title: Images
 ---
 ## Representing humans
-In order to offer more diverse and expressive ways to represent humans, we created a custom set of Open Peeps based on [the work of Pablo Stanley](https://www.openpeeps.com/). These are great for showing emotion and to represent anonymous user groups.
+
+In order to represent a more diverse spectrum of human experiences, we’ve adopted an illustration style called *18F Folks*. 
```

Comment:
> we’ve adopted an illustration style called *18F Folks*.

This line reads a little funny to me. I think 18F Folks refers to the customizable library, not necessarily the style. Much in the same way Open Peeps itself is called a library and not a style – even if it clearly uses a particular and consistent style.  So my suggestion  is something more like "We've created a library called "18F Folks, based on Pablo Stanley's..."